src/content/concepts/index.md

```diff
@@ -114,6 +114,8 @@ The configuration above has defined a `rules` property for a single module with
 
 W> It is important to remember that when defining rules in your webpack config, you are defining them under `module.rules` and not `rules`. For your benefit, webpack will warn you if this is done incorrectly.
 
+W> Keep in mind that when using regex to match files, you may not quote it. i.e `/\.txt$/` is not the same as `'/\.txt$/'`/ `"/\.txt$/"`. The former instructs webpack to match any file that ends with .txt and the later instructs webpack to match a single file with an absolute path '.txt'; this is likely not your intention. 
```

Comment:
on one hand this is regex knowledge related warning, on the other it may be useful to users. i am not voting for this and also not against it. If you are good with this pls merge @montogeek